src/webui/www/private/download.html

```diff
@@ -21,7 +21,7 @@ <h2 class="vcenter">QBT_TR(Download Torrents from their URLs or Magnet links)QBT
             <fieldset class="settings" style="border: 0; text-align: left;">
                 <div class="formRow" style="margin-top: 6px;">
                     <label for="savepath" class="leftLabelLarge">QBT_TR(Save files to location:)QBT_TR[CONTEXT=HttpServer]</label>
-                    <input type="text" id="savepath" name="savepath" style="width: 16em;" />
+                    <input type="text" id="savepath" name="savepath" autocorrect="off" autocapitalize="none" style="width: 16em;" />
```

Comment:
Can you move these attributes to the file's `<form>` element so that they apply to all inputs? Same for `upload.html`.

In the future we should probably put `preferences_content.html` inside a form element so that we can easily cover all inputs.

Reply:
@Piccirello, apparently, the author abandoned it... What are we supposed to do?

Reply:
> @Piccirello, apparently, the author abandoned it... What are we supposed to do?

I have been busy maybe this weekend I will do the changes, I am happy for anyone else to also implement them.